test/integration/rules/runner.js

```diff
@@ -95,7 +95,11 @@
               // check that all nodes are removed
               assert.equal(JSON.stringify(targets), '[]');
             } else {
-              assert(false, 'there are no ' + collection);
+              assert.lengthOf(
+                test[collection],
+                0,
```

Comment:
This lets us say not to expect any violations. Previously that was just erroring.